docs/victoriametrics-cloud/account-management/registration-and-trial.md

```diff
@@ -0,0 +1,36 @@
+---
+weight: 1
+title: Registration and Trial
+disableToc: true
+menu:
+  docs:
+    parent: account-management
+    weight: 1
+---
+
+Start your registration process by visiting the [Sign Up](https://console.victoriametrics.cloud/signUp?utm_source=website&utm_campaign=docs_registration_and_trial) page.
+
+## Authentication methods
+
+VictoriaMetrics Cloud supports user registration and authentication via the following mechanisms:
+
+1. Sign up with Google.
+2. Email and password.
+
+
+## Trial period and credits
+
+After registration, every new user is granted with $200 in credits to spend during the trial period.
+The trial period starts once the first deployment is created and lasts for 30 days.
+
+In general, VictoriaMetrics Cloud billing is based on the time a deployment is running.
```

Comment:
running or stopped, so if you provision it we charge, if it;s stopped only for storage

Reply:
```suggestion
In general, VictoriaMetrics Cloud billing is based on the time a deployment consumes resources (stopped deployments consume storage).
```